src/components/Forecast/Widget.jsx

```diff
@@ -0,0 +1,181 @@
+import React from 'react'
+import Form from './Form/Form'
+import WeatherDetailsToday from './WeatherDetailsToday/WeatherDetailsToday'
+import FiveDayForecast from './FiveDayForecast/FiveDayForecast'
+
+import './Widget.scss'
+
+const myAPIKey = 'dce6d076b7a2ffe64e08efe5ff779fa3'
```

Comment:
I appreciate how easy it was for me to start and test the app because of this. But I'd really advise you to NEVER add API keys in your code and push it to the repo. Not only is it terrible for maintenance, but you can always be sure someone is actively scraping GitHub for API keys and login credentials they can abuse.

Instead op posting it in the code, use environment variables, they're great for stuff like this!

Read: https://12factor.net/config